packages/js/src/ui/config/default-appearance.ts

```diff
@@ -0,0 +1,13 @@
+import { Variables } from '../context';
+
+export const defaultVariables: Required<Variables> = {
+  colorPrimary: '#0081F1',
+  colorPrimaryForeground: 'white',
+  colorSecondary: '#F3F3F3',
+  colorSecondaryForeground: '#1A1523',
+  colorBackground: '#FFFFFF',
+  colorForeground: '#1A1523',
+  colorNeutral: 'black',
+  fontSize: 'inherit',
+  borderRadius: '0.375rem',
+};
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
❓ **question:** ‏@desiprisg How will we handle the hover color and icon colors

Reply:
Icon colors would be a shade of `foreground`. Hover is going to be a shade of `colorNeutralAlpha` for dropdowns and shades of the underlying color for buttons.